build.gradle.kts

```diff
@@ -4,6 +4,7 @@ group = "org.tree-ware"
 version = "1.0-SNAPSHOT"
 
 val mySqlConnectorVersion = "8.0.27"
+val tcVersion = "1.17.2"
```

Comment:
nit: "tc" is not a common/well-known abbreviation, so please expand it to "test containers"